### DISCUSSION TITLE
user group / active directory

### DISCUSSION THREAD

**Discussion by antoinebio** [requarks/wiki#7483] [Error / Bug Report]
Hello :
We've set up Active Directory strategy on our WIKI using wikijs framework, so that my "company collaborator" access the wiki with their own enterprise authentication account info.

It works without error when testing with our collaborator (no red error message, green and success login message instead...)

In the LDAP Strategy setup, We've included auto-registration option to assign the user that is login to an existing Group.

<img width="737" alt="image" src="https://github.com/user-attachments/assets/944c8271-2b6f-4626-a145-6354878c7ef3" />

However after initial test (ie. first login of one of that user) , I am facing that kind of page :

<img width="371" alt="image" src="https://github.com/user-attachments/assets/7a0c89a6-de9c-470a-a0f3-5a2612958ccc" />

So in the admin panel, I go to user panel, where I can see that the "company collaborator" has tried to connect.
If I go and edit his profil, I can force that group assignment again (wierd that the LDAP strategy wasn't working fine...).

So I force his assignment.

<img width="500" alt="image" src="https://github.com/user-attachments/assets/47aae413-4f98-4920-b50c-39895501f52f" />

Then I ask again my colleague to login.... still He is been rejected to that page

<img width="371" alt="image" src="https://github.com/user-attachments/assets/7a0c89a6-de9c-470a-a0f3-5a2612958ccc" />

And up to most, If I check again his profil I can see that the group assigment was reset to defaut (without taking into account manual modification I did before...).

Do you know how I can face out that conflict ?

this is where I do not catch why as an Admin when I force user profil assignment to a Group it being reset after user login attempt...

thanks.

![image](https://github.com/user-attachments/assets/e900f60a-da01-44fc-9578-b806a6f4d196)

**Comment by AlexJFox** [comment]
This looks more like a page rule (Groups > Page Rules) error than an authentication error

**Reply by antoinebio** [reply]
Hi @AlexJFox 
thanks for the reply.

what kind of rule should I edit ?

<img width="1007" alt="image" src="https://github.com/user-attachments/assets/173000fa-08eb-44c5-bec4-b81329318fce" />

<img width="275" alt="image" src="https://github.com/user-attachments/assets/50cc584f-4db6-4333-8550-c1d6ce687034" />

<img width="274" alt="image" src="https://github.com/user-attachments/assets/7dc1b3ff-fb75-4646-a3ff-7a95b9d442bc" />
